sentry-android-core/src/main/java/io/sentry/android/core/LoadClass.java

```diff
@@ -9,10 +13,34 @@
    * Try to load a class via reflection
    *
    * @param clazz the full class name
-   * @return a Class<?>
-   * @throws ClassNotFoundException if class is not found
+   * @param logger an instance of ILogger
+   * @return a Class<?> if it's available, or null
    */
-  public @NotNull Class<?> loadClass(@NotNull String clazz) throws ClassNotFoundException {
-    return Class.forName(clazz);
+  public @Nullable Class<?> loadClass(final @NotNull String clazz, final @Nullable ILogger logger) {
+    try {
+      return Class.forName(clazz);
+    } catch (ClassNotFoundException e) {
+      if (logger != null) {
+        logger.log(SentryLevel.WARNING, "Failed to load class " + clazz, e);
```

Comment:
I feel this log as `Warning` with a message saying `Failed to` will be a source of confusion to users.
This would be the right behavior whether we were "really trying to find it" but looking from the point of view of `isClassAvailable`, it sounds like just a check: "is it there? It's OK if not..".

Another way to look at this: Do we expect the class to be found in >95% of cases? I believe for timber for example the answer is _no_ so best to use log level Debug here and let the caller decide whether that's a warning or not.

```suggestion
        logger.log(SentryLevel.DEBUG, "Class not available:" + clazz, e);
```